source/_integrations/home_connect.markdown

```diff
@@ -807,6 +807,13 @@ This sensor will be available only if the program is running
     - **Map 3**: `map3`
 
     </details>
+- **Current cavity temperature**:
+  - **Description**: Represents the current cavity temperature.
+  - **Availability**: Oven
+
+{% important %}
+It is not recommended to use the **Current cavity temperature** sensor because the temperature might not provide the necessary accuracy.
```

Comment:
Not recommended for what? To measure the core temp of meat?

Reply:
I don't know 😅, I copy-pasted the information from the docs

Reply:
Also, there is no more information in the docs about what does the entity do

Reply:
I see. Those docs aren't very helpful there. Maybe we remove the notice? It isn't really helpful if we don't know what it actually measures and what we should use instead.

Reply:
Adding the disclaimer is something that @MartinHjelmare and I talked about adding because we wanted to make sure that the disclaimer from the Home Connect documentation was added here too.
About what measures, the name of the key (`Cooking.Oven.Status.CurrentCavityTemperature`) makes me think that it is a sensor that an Oven can have that measures the air temperature of the cavity,

Reply:
My interpretation of "oven cavity" is it's the inside compartment of the oven where you cook the food.

In the API provider docs, this notice is mentioned in the context of pre-heating an oven. This data point is also left out of the table of available data points and only mentioned in this note. My impression is they don't want to promote this data point since its precision is low. That's just a guess though.